server/app/routes/order/index.js

```diff
@@ -0,0 +1,61 @@
+'use strict';
+
+const path = require('path');
+const router = require('express').Router();
+const db = require(path.join(__dirname, '../../../db'));
+const Song = db.model('song');
+const Order = db.model('order');
+const Address = db.model('address');
+const stripe = require("stripe")("sk_test_BQokikJOvBiI2HlWgH4olfQ2");
+const Promise = require('sequelize').Promise;
+
+router.get('/', function (req, res, next){
+  
+   
+});
+
+router.post('/', function (req, res, next) {
```

Comment:
@jonperrelle @kmasila00 let's walk through what's happening here. maybe add some descriptive comments?
(note to self: sync with #109 )